client/src/components/_common/HighlightSearchTerm/HighlightSearchTerm.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './HighlightSearchTerm.module.scss';
+
+const HighlightSearchTerm = ({ searchTerm, content }) => {
+  const highlightParts = () => {
+    const parts = content.split(new RegExp(`(${searchTerm})`, 'gi'));
+    return parts.map((part, i) =>
+      part.toLowerCase() === searchTerm.toLowerCase() ? (
```

Comment:
I changed logic to use `match` instead of `toLowerCase`. Tried the other suggestion to use `.replace()` function instead but couldn't get it to work properly. I am getting [object] [object' inside the string that needs to be highlighted. Is it because .replace() isn't good for returning the <b>...</b> and only good for string? Let me know.